mitiq/pec/representations/tests/test_depolarizing.py

```diff
@@ -50,25 +50,13 @@
 
 
 def single_qubit_depolarizing_overhead(noise_level: float) -> float:
```

Comment:
This function is not picked up by the API doc because it does not have the Args / Return google-style formatting.